element/test/index.js

```diff
@@ -1,86 +1,81 @@
 /**
  * External dependencies
  */
-import { expect } from 'chai';
-
-/**
- * Internal dependencies
- */
 import { createElement, renderToString, concatChildren, switchChildrenNodeName } from '../';
```

Comment:
I'm wondering if it would be possible to add Eslint rule that would verify such use case.